apps/mobile/src/libs/components/wheel/libs/components/animated-pulsing-sector/animated-pulsing-sector.tsx

```diff
@@ -0,0 +1,115 @@
+import { Animated, Circle } from "~/libs/components/components";
+import { type SectorInitialSharedValues } from "~/libs/types/types";
+
+import { getRandomValue, getSectorParameters } from "../../helpers/helpers";
+import { useWheelAnimation } from "../../hooks/hooks";
+
+type Properties = {
+	animationRepetitions: number;
+	animationTime: number;
+	centerGap: number;
+	centerPoint: number;
+	endPercentInner: number;
+	endPercentOuter: number;
+	height: number;
+	layerOffset: number;
+	maxHeight: number;
+	outlineColor: string;
+	sectorColor: string;
+	startPercentInner: number;
+	startPercentOuter: number;
+};
+
+const HALF_VALUE_COEFFICIENT = 0.5;
+
+const AnimatedCircle = Animated.createAnimatedComponent(Circle);
+
+const AnimatedPulsingSector: React.FC<Properties> = ({
+	animationRepetitions,
+	animationTime,
+	centerGap,
+	centerPoint,
+	endPercentInner,
+	endPercentOuter,
+	height,
+	layerOffset,
+	maxHeight,
+	outlineColor,
+	sectorColor,
+	startPercentInner,
+	startPercentOuter,
+}: Properties) => {
+	const {
+		innerDashArrayDash,
+		innerDashArrayGap,
+		innerDashOffset,
+		innerStrokeWidth,
+		outerDashArrayDash,
+		outerDashArrayGap,
+		outerDashOffset,
+		outerStrokeWidth,
+		radius,
+	} = getSectorParameters({
```

Comment:
You already use `getSectorParameters` inside of a hook, do we need to have it here as well?

Reply:
I need it to calculate initial values for AnimatedPulsingSector, and then pass them as initialSharedValues.
I can omit using getSectorParameters for initial values and instead set them to 0, but this way the loader won't be present on the page at once. It will appear and then continues pulsing.
Inside the hook it is called to update sharedValues.